<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Course Helper</string>
  </property>
  <widget class="QWidget" name="_centralWidget">
   <layout class="QVBoxLayout" name="_centralWidgetLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTabWidget" name="_tabWidget">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="_courseTab">
       <attribute name="title">
        <string>Listes de courses</string>
       </attribute>
       <layout class="QHBoxLayout" name="_courseTabLayout">
        <item>
         <layout class="QVBoxLayout" name="_courseLayout1">
          <item>
           <widget class="QTreeWidget" name="courseTreeWidget">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideRight</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
            <property name="columnCount">
             <number>1</number>
            </property>
            <attribute name="headerCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="headerHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>  Listes de courses</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="_courseLayout3">
            <item>
             <widget class="QLabel" name="_courseLabelFind">
              <property name="text">
               <string>Rechercher : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="courseLineEditFind"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="_courseLine">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="_courseLayout2">
          <item>
           <widget class="QPushButton" name="courseAddButton">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="courseModifyButton">
            <property name="text">
             <string>Modifier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="courseDeleteButton">
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="_courseVS">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="courseCheckButton">
            <property name="text">
             <string>Consulter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_recetteTab">
       <attribute name="title">
        <string>Recettes</string>
       </attribute>
       <layout class="QHBoxLayout" name="_recetteTabLayout">
        <item>
         <layout class="QVBoxLayout" name="_recetteLayout2">
          <item>
           <widget class="QTreeWidget" name="recetteTreeWidget">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerItem</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="headerCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>  Listes de recettes</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="_recetteLayout3">
            <item>
             <widget class="QLabel" name="_recetteLabelFind">
              <property name="text">
               <string>Rechercher : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="recetteLineEditFind"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="_recetteLine">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="_recetteLayout1">
          <item>
           <widget class="QPushButton" name="recetteAddButton">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="recetteModifyButton">
            <property name="text">
             <string>Modifier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="recetteDeleteButton">
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="_recetteVS">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_productTab">
       <attribute name="title">
        <string>Produits</string>
       </attribute>
       <layout class="QHBoxLayout" name="_productTabLayout">
        <item>
         <layout class="QVBoxLayout" name="_productLayout1">
          <item>
           <widget class="QTableView" name="productTableView">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="_productLayout3">
            <item>
             <widget class="QLabel" name="_productLabelFind">
              <property name="text">
               <string>Rechercher : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="produitLineEditFind"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="_produitLine">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="_produitLayout2">
          <item>
           <widget class="QPushButton" name="productAddButton">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="productModifyButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Modifier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="productDeleteButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="_produitVS">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="toolTip">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>productAddButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_addProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>productModifyButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_modifyProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>productDeleteButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_deleteProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>productTableView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_modifyProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_addProduct()</slot>
  <slot>on_modifyProduct()</slot>
  <slot>on_deleteProduct()</slot>
 </slots>
</ui>
